book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative './rentals'
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
```

Comment:
- [ ] Please make sure the rentals variable is readable and writable to work for the associations as expected,  To achieve this, You can add that in the accessor.